### PR TITLE
Do not control SqlEditor height

### PR DESCRIPTION
**Pull request by mshustov**
closes https://github.com/grafana/clickhouse-datasource/issues/684

There is a bug in logic that on every change in the editor calls `saveChanges({expand:true})`, which resets the `rawSql` value back to the original state.
Instead of fixing this bug, I checked how Grafana handles this logic itself. From [this example](https://github.com/grafana/grafana/blob/f59e76d54b0f5ea11e23f2f59eb3281682c7d1d0/public/app/plugins/datasource/tempo/traceql/TraceQLEditor.tsx#L81), it looks like the built-in Grafana editor handles height updates inside `CodeEditor` component, so I removed the external logic to manage component height.

### Changelog
Remove manual expansion of SqlEditor.

### CODE REVIEW

File: src/components/SqlEditor.tsx [hunk 1]
```diff
@@ -16,22 +15,25 @@ import { pluginVersion } from 'utils/version';
...
-  on: boolean;
+function setupAutoSize(editor: monacoTypes.editor.IStandaloneCodeEditor) {
+  const container = editor.getDomNode();
+  const updateHeight = () => {
+    if (container) {
+      const contentHeight = Math.max(100, Math.min(1000, editor.getContentHeight()));
+      const width = parseInt(container.style.width, 10);
+      container.style.width = `${width}px`;
+      container.style.height = `${contentHeight}px`;
+      console.log('contentHeight', contentHeight, 'width', width);
```
Comment (asimpson) [inline src/components/SqlEditor.tsx:26]:
```suggestion
```
nit: we don't need this right?

**Review by asimpson — CHANGES_REQUESTED** [review]
:+1: looks good to me, thanks!

Just one nit and I'll approve.

**Review by asimpson — APPROVED** [review]

**Comment by asimpson** [comment]
@mshustov Looks like it. The build passed though which is what I care about. Can you merge `main` and I can merge the PR?

**Comment by mshustov** [comment]
@asimpson could you merge?

**Pull request merged**